lib/utils/getHoldingLocations.js

```diff
@@ -0,0 +1,99 @@
+import {
+  HOLDINGS_API,
+  LIMIT_MAX,
+  LOCATIONS_API,
+} from '../constants';
+import { batchRequest } from './batchFetch';
+import { extendKyWithTenant } from './extendKyWithTenant';
+
+const DEFAULT_DATA = [];
+
+export const getHoldingLocations = async ({
```

Comment:
Based on the name of this function, I expect to get a list of locations for a specific holding according to certain parameters, not for a list of holdings.